trustfall_core/src/graphql_query/mod.rs

```diff
@@ -1,3 +1,4 @@
+//! Translating GraphQL queries to Rust types
```

Comment:
Perhaps a bit more "signposting in global coordinates" here? Maybe something like:
```suggestion
//! Parsing Trustfall queries into Rust types,
//! which are then handed to the frontend for further processing.
```
What do you think?